sigs.yaml

```diff
@@ -1165,6 +1154,17 @@ sigs:
       url: https://zoom.us/j/91929881559?pwd=WllxazhTUzBFN1BNWTRadTA3NGtQQT09
       archive_url: https://docs.google.com/document/d/10aTeq2lhXW_3aFQAd_MdGjY8PtZPslKhZCCcXxFp3_Q/edit
       recordings_url: https://www.youtube.com/playlist?list=PL69nYSiGNLP29D0nYgAGWt1ZFqS9Z7lw4
+  - name: cluster-api-provider-oci
+    owners:
+    - https://github.com/oracle/cluster-api-provider-oci/blob/main/OWNERS
```

Comment:
why is this provider referencing an external repo?
it must be under kubernetes/kubernetes-sigs to be in the sigs.yaml file.

Reply:
We are in the process of donating it https://github.com/kubernetes/org/issues/3404

This change was accepted as part of https://github.com/kubernetes/community/pull/6855. This new PR is simply moving it from the `Cloud-Provder` section into the `Cluster-Lifecycle` section.

Reply:
i am surprised this was previously accepted, since TMK a project must first be in our orgs.

cc @jberkus 
could you LGTM/approve again?

Reply:
/assign @jdumars
hi, could you please help us figuring out if external repositories can be subprojects. afaik, they cannot.